### PR TITLE
Dịch "autorec_vn" - Phần 3

### LINKED ISSUE #2683: Dịch "autorec_vn" - Phần 3
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_recommender-systems/autorec_vn.md#L153-L247

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by trGiang99**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Điền số issue sau dấu `#` (không có dấu cách): Close #2683

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy thảo luận thêm ở thẻ Discussions của dự án.

* [x] Format của bản gốc có được giữ nguyên không?
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn. Bạn đã kiểm tra rằng mỗi đoạn văn đã nằm một dòng?

* [x] Điền tên đầy đủ của bạn vào mục **Những người thực hiện** ở cuối tập tin.

## Lưu ý cho người phản biện (Reviewer)

1. Chỉ review khi Pull Request (PR) này đã sẵn sàng cho việc review. Một PR sẵn sàng khi PR đó không có label `status: wip`.
2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

### CODE REVIEW

File: chapter_recommender-systems/autorec_vn.md [hunk 1]
```diff
@@ -241,7 +244,9 @@ d2l.train_recsys_rating(net, train_iter, test_iter, loss, trainer, num_epochs,
 * Can you find a better combination of decoder and encoder activation functions?
 -->
 
-*dịch đoạn phía trên*
+* Thay đổi kích thước ẩn của AutoRec để quan sát ảnh hưởng của việc này lên hiệu năng mô hình.
+* Hãy thử thêm vào nhiều tầng ẩn. Việc này có giúp cải thiện hiệu năng mô hình?
```
Comment (nguyenthaibinh) [inline chapter_recommender-systems/autorec_vn.md:248]:
```suggestion
* Hãy thử thêm vào nhiều tầng ẩn. Việc này có giúp cải thiện hiệu năng mô hình không?
```
```suggestion
* Hãy thử thêm vào nhiều tầng ẩn. Việc này có giúp cải thiện hiệu năng mô hình?
```

File: chapter_recommender-systems/autorec_vn.md [hunk 2]
```diff
@@ -156,14 +156,14 @@ class AutoRec(nn.Block):
...
-## *dịch tiêu đề trên*
+## Lập trình lại Phép đánh giá
 
 
 <!--
 Since the input and output have been changed, we need to reimplement the evaluation function, while we still use RMSE as the accuracy measure.
 -->
 
-*dịch đoạn phía trên*
+Do đầu vào và đầu ra thay đổi nên ta cần phải lập trình lại hàm đánh giá, trong khi đó ta vẫn sẽ sử dụng RMSE làm phép đo độ chính xác.
```
Comment (cuongvng) [inline chapter_recommender-systems/autorec_vn.md:166]:
```suggestion
Do đầu vào và đầu ra thay đổi nên ta cần phải lập trình lại hàm đánh giá, nhưng vẫn sử dụng RMSE làm phép đo độ chính xác.
```

**Review by cuongvng — APPROVED** [review]

**Review by duythanhvn — APPROVED** [review]
